types/mercadopago/models/customers/create-payload.model.d.ts

```diff
@@ -3,6 +3,8 @@ import { Identification } from '../../shared/payer-identification';
 import { Phone } from '../../shared/phone';
 
 export interface CreateCustomerPayload {
+  /**Email do cliente */
+  email:string
```

Comment:
Sorry, I forgot to set the parameter to optional. Can you change to `email?:`, please?

Reply:
done